plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -468,6 +525,13 @@ public Optional<ConstraintApplicationResult<ConnectorTableHandle>> applyFilter(
         return Optional.of(new ConstraintApplicationResult<>(updatedHandle, constraint.getSummary(), false));
     }
 
+    private String getRemoteSchemaName(BigQueryClient client, String projectId, String datasetName)
```

Comment:
Probably we should also call this in `createSchema` to avoid creating schemas in BigQuery which differ only in case. Separate and pre-existing issue however.

Reply:
Turns out this is not needed/the wrong thing to do. When creating schemas we already list them and the listing will include the lowercase name so Trino will see a schema already exists - I've added a test to verify this similar to what you added for DROP SCHEMA in https://github.com/trinodb/trino/pull/13812